### PR TITLE
Add Zulip badge and acknowledgement

### PR DESCRIPTION
**Pull request by vesalvojdani**
It would be polite to [acknowledge](https://zulip.com/help/linking-to-zulip-website) the zulip standard hosting, and somewhat related, I think we should make the chat public, so that our endless end-users can ask for help there. I only wonder if it would be better to create a dedicated support channel and link there. 

Anyway, I think our Zulip instance is open for the public already, but we may want to check that everything that should be private there is set as private before advertising the URL more broadly. Then, we can also check the box to allow them to list it in the [communities directory](https://zulip.com/help/communities-directory).

### CODE REVIEW

**Comment by michael-schwarz** [comment]
We might want to lock down `#random` in this case. There's a lot of tongue-in-cheek things there which I think is absolutely fine for students or researchers in Freiburg or at LMU and so on to see, but maybe not so much for the  entire world.

File: README.md [hunk 1]
```diff
@@ -56,3 +57,9 @@ docker run -it --rm -v $(pwd):/data goblint /data/tests/regression/04-mutex/01-s
 If pulled from GitHub Container Registry, use the container name `ghcr.io/goblint/analyzer:latest` (or `:nightly`) instead.
 
 For further information, see [documentation](https://goblint.readthedocs.io/en/latest/user-guide/running/).
+
+## Acknowledgements
+
+Work on Goblint was supported in part by Deutsche Forschungsgemeinschaft (DFG) (47140942/1480 [PUMA](https://gepris.dfg.de/gepris/projekt/4714094), 378803395/2428 [ConVeY](http://convey.in.tum.de)), ARTEMIS Joint Undertaking (269335 [MBAT](http://www.mbat-artemis.eu/home/)), ITEA3 project 14014 [ASSUME](http://assume-project.eu/), and the Estonian Research Council ([IUT2-1](https://www.etis.ee/Portal/Projects/Display/561b7b1d-d1dd-43a2-90e5-0661de823823?lang=ENG), [PSG61](https://www.etis.ee/Portal/Projects/Display/743243bb-15c2-47b3-9c10-e7d86a9a276d?lang=ENG)).
```
Comment (michael-schwarz) [inline README.md:63]:
```suggestion

Work on Goblint was supported in part by Deutsche Forschungsgemeinschaft (DFG) (47140942/1480 [PUMA](https://gepris.dfg.de/gepris/projekt/4714094), 378803395/2428 [ConVeY](http://convey.in.tum.de)), ARTEMIS Joint Undertaking (269335 [MBAT](http://www.mbat-artemis.eu/home/)), ITEA3 project 14014 [ASSUME](http://assume-project.eu/), the Shota Rustaveli National Science Foundation of Georgia [FR-21-7973](https://viam.science.tsu.ge/new/index.php?lang=eng&page=projects&subpage=111), the Estonian Research Council ([IUT2-1](https://www.etis.ee/Portal/Projects/Display/561b7b1d-d1dd-43a2-90e5-0661de823823?lang=ENG), [PSG61](https://www.etis.ee/Portal/Projects/Display/743243bb-15c2-47b3-9c10-e7d86a9a276d?lang=ENG)) and the Estonian Centre of Excellence in IT (EXCITE), funded by the European Regional Development Fund.
```

would be what we have if we want to be consistent with the website.

**Comment by sim642** [comment]
> Anyway, I think our Zulip instance is open for the public already,

I thought it was open just to UniTartu and TUM email addresses or something?

I'm not sure there's much value in promoting it as a support channel. GitHub issues have worked fine and quickly enough in cases we've had outside interest.

**Comment by michael-schwarz** [comment]
What is the plan here?

**Comment by sim642** [comment]
> > Anyway, I think our Zulip instance is open for the public already,
> 
> I thought it was open just to UniTartu and TUM email addresses or something?

What is the case here then?


> We might want to lock down `#random` in this case. There's a lot of tongue-in-cheek things there which I think is absolutely fine for students or researchers in Freiburg or at LMU and so on to see, but maybe not so much for the entire world.

Because it's not limited to UniTartu, TUM, etc. domains, then we still have a number of public channels that everyone could see if they just bothered to log in.

**Comment by michael-schwarz** [comment]
> What is the case here then?

It's not limited.

> Because it's not limited to UniTartu, TUM, etc. domains, then we still have a number of public channels that everyone could see if they just bothered to log in.

Yes, but I don't think there's anything bad in any of these. Also #ramom seems to be OK after having a closer look.

**Pull request merged**